src/assisted_test_infra/test_infra/helper_classes/cluster.py

```diff
@@ -1292,3 +1293,79 @@ def wait_and_kill_installer(self, host):
         # Kill installer to simulate host error
         selected_node = self.nodes.get_node_from_cluster_host(host)
         selected_node.kill_installer()
+
+    def format_host_current_network(self):
```

Comment:
```suggestion
    def format_host_current_network(self) -> dict[str, dict[str, list[str]]]:
```
something like this I guess :thinking: